test/src/main/resources/scala/test/more-hellos

```diff
@@ -0,0 +1,2 @@
+More Hello
```

Comment:
It seems none of the files in `test/src/main/resources/scala/test/` are exported via `exports_files`, they were referenced directly (https://github.com/bazelbuild/rules_scala/blob/master/test/BUILD#L132-L133). Ill add a new BUILD file to export them too